components/match2/wikis/apexlegends/match_summary.lua

```diff
@@ -265,7 +265,7 @@ local GAME_STANDINGS_COLUMNS = {
 ---@param args table
 ---@return string
 function CustomMatchSummary.getByMatchId(args)
-	local match = MatchGroupUtil.fetchMatchForBracketDisplay(args.bracketId, args.matchId)
+	local match = MatchGroupUtil.fetchMatchForBracketDisplay(args.bracketId, args.matchId) --[[@as table]]
```

Comment:
same

Reply:
because we are adding stuff into match/games in the lines below
![Screenshot 2024-06-12 161231](https://github.com/Liquipedia/Lua-Modules/assets/75081997/e120b6e5-9453-45c8-a282-792ebaedc081)

alternative would be to define a custom classes there

```lua
---@class ApexMatchGroupUtilMatch: MatchGroupUtilMatch
---@field scoringTable {kill: number, placement: {rangeStart: integer, rangeEnd: integer, score: number}[]}
---@games ApexMatchGroupUtilGame[]

---@class ApexMatchGroupUtilGame: MatchGroupUtilGame
---@field scoringTable {kill: number, placement: {rangeStart: integer, rangeEnd: integer, score: number}[]}
---@field stream table
```

Reply:
Reported it at https://github.com/LuaLS/lua-language-server/issues/2710